sourcegraph/sourcegraph
# (feat) emphasize cody for new users on dotcom

Closes https://github.com/sourcegraph/sourcegraph/issues/51399.

This PR:
- feat(SearchPageContent.tsx): add CodySignUpCtaSection to dotcom search page for unauthenticated users
- feat(BlobPage.tsx): add TryCodyWidget to dotcom blob page for authenticated users
  - feat(TryCodyWidget.tsx): make it auto-dismissible whenever Cody on web is tried
- feat(CloudSignUpPage.tsx): update feature list and page background
- refactor(GlobalNavbar.tsx): change "About" link text to "About Sourcegraph"
- refactor(SearchPageContent.tsx): update search page content message to reflect new features

## Test plan
- `sg start dotcom`
- Check the search home page
- Sign in and go to any blob page

## Demo

https://user-images.githubusercontent.com/6717049/236134788-642c45c3-8149-4945-b3b6-4c936d229c09.mov
